Source/WTF/wtf/PrintStream.cpp

```diff
@@ -149,6 +149,12 @@ void printInternal(PrintStream& out, char16_t value)
     out.printf("%lc", value);
 }
 
+void printInternal(PrintStream& out, char32_t value)
+{
+    // Print each char32_t as an integer.
```

Comment:
This may be something we want to revisit.  I think it may be slightly better to effectively call printInternal(char16_t) with the high bits if they're nonzero then printInternal(char16_t) with the low bits.